specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-11-15-preview/managedCassandra.json

```diff
@@ -774,657 +693,1717 @@
+              "type": "string",
+              "description": "Key uri to use for encryption of managed disks. Ensure the system assigned identity of the cluster has been assigned appropriate permissions(key get/wrap/unwrap permissions) on the key."
             },
-            "externalSeedNodes": {
-              "type": "array",
-              "description": "List of IP addresses of seed nodes in unmanaged data centers. These will be added to the seed node lists of all managed nodes.",
-              "items": {
-                "$ref": "#/definitions/SeedNode"
-              }
+            "backupStorageCustomerKeyUri": {
+              "type": "string",
+              "description": "Indicates the Key Uri of the customer key to use for encryption of the backup storage account."
             },
-            "seedNodes": {
-              "readOnly": true,
-              "type": "array",
-              "items": {
-                "$ref": "#/definitions/SeedNode"
-              },
-              "description": "List of IP addresses of seed nodes in the managed data centers. These should be added to the seed node lists of all unmanaged nodes."
+            "sku": {
+              "type": "string",
+              "description": "Virtual Machine SKU used for data centers. Default value is Standard_DS14_v2"
             },
-            "hoursBetweenBackups": {
+            "diskSku": {
+              "type": "string",
+              "description": "Disk SKU used for data centers. Default value is P30."
+            },
+            "diskCapacity": {
               "type": "integer",
               "format": "int32",
-              "description": "Number of hours to wait between taking a backup of the cluster. To disable backups, set this property to 0."
+              "description": "Number of disk used for data centers. Default value is 4."
             },
-            "deallocated": {
+            "availabilityZone": {
               "type": "boolean",
-              "description": "Whether the cluster and associated data centers has been deallocated."
+              "description": "If the data center has Availability Zone support, apply it to the Virtual Machine ScaleSet that host the cassandra data center virtual machines."
             },
-            "cassandraAuditLoggingEnabled": {
+            "deallocated": {
               "type": "boolean",
-              "description": "Whether Cassandra audit logging is enabled"
+              "description": "Whether the data center has been deallocated."
+            },
+            "provisionError": {
+              "$ref": "#/definitions/CassandraError",
+              "description": "Error related to resource provisioning."
+            },
+            "authenticationMethodLdapProperties": {
+              "$ref": "#/definitions/AuthenticationMethodLdapProperties"
+            }
+          }
+        }
+      }
+    },
+    "CassandraClusterRepairPublicProperties": {
```

Comment:
>CassandraClusterRepairPublicProperties [](http://example.com/codeflow?start=5&length=38)

I think it would be easy to avoid using trademarks and brand names in typenames here.